django_lightweight_queue/app_settings.py

```diff
@@ -36,3 +36,5 @@ def setting(suffix, default):
 ENABLE_FLUENT_LOGGING = setting('ENABLE_FLUENT_LOGGING', False)
 FLUENT_HOST = setting('FLUENT_HOST', '127.0.0.1')
 FLUENT_PORT = setting('FLUENT_PORT', 24224)
+
+ATOMIC_JOBS = setting('ATOMIC_JOBS', True)
```

Comment:
Would `ATOMIC_JOBS_BY_DEFAULT` be a better name? or something else that implies it can be overridden per job?

Reply:
I went for this name by analogy with Django's built-in `ATOMIC_REQUESTS`, do you think this is different enough to warrant changing?